File: code/server/src/Core/Entities/Componente.cs
using IMT_Reservas.Server.Core.Abstraction;

namespace IMT_Reservas.Server.Core.Entities;

public class Componente : Entity
{
    public string Nombre { get; set; } = string.Empty;
    public string? Descripcion { get; set; }
    public string Modelo { get; set; } = string.Empty;
    public string? Tipo { get; set; }
    public double? PrecioReferencia { get; set; }
    public int IdEquipo { get; set; }
    public string? UrlDataSheet { get; set; }
}

File: code/client/src/app/entities/admin/model/componente.ts
import { BaseModel } from '@shared/model';
export class Componente extends BaseModel {
  Nombre: string | null = null;
  Modelo: string | null = null;
  Tipo: string | null = null;
  Descripcion?: string | null = null;
  PrecioReferencia?: number | null = null;
  NombreEquipo?: string | null = null;
  CodigoImtEquipo: string | null = null;
  UrlDataSheet?: string | null = null;
  IdEquipo?: number | null = null;
}
